src/inverse-kinematics/tests/ConvexHullHelpersUnitTest.cpp

```diff
@@ -97,19 +97,37 @@ void testConvexHullProjectionWithGravity()
     // Note: positive margin means inside, negative outside
     ASSERT_IS_TRUE(projectionConstraint.computeMargin(projectionConstraint.project(testPoint)) < 0);
 
-    // If the direction along with we are projecting is the normal of the plane, the projection
+    // If the direction along witch we are projecting is the normal of the plane, the projection
     // should match the one done without direction
-    iDynTree::Direction planeNormal(0.0, 0.0, -1.0);
+
+    //iDynTree::Direction planeNormal(0.0, 0.0, -1.0);
     //ASSERT_EQUAL_VECTOR(projectionConstraint.project(testPoint), projectionConstraint.projectAlongDirection(testPoint, planeNormal));
+    iDynTree::Vector3 planeNormalVector;
+    planeNormalVector.setVal(0, 0.0);
+    planeNormalVector.setVal(1, 0.0);
+    planeNormalVector.setVal(2, -1.0);
```

Comment:
This is indeed a direction, so it is ok to use `iDynTree::Direction` for it.

Reply:
I didn't notice that the class `Direction` derives from `Vector3`. Fixed!